tests/fixtures/mock_spoc.py

```diff
@@ -44,12 +44,15 @@
 
 mock_collection_spoc_2 = deepcopy(mock_spoc_2)
 mock_collection_spoc_2["collection_title"] = "Best of the Web"
+mock_collection_spoc_2["sponsored_by_override"] = "Brought to you by WallMarket"
 mock_collection_spoc_3 = deepcopy(mock_spoc_3_cta)
 mock_collection_spoc_3["collection_title"] = "Best of the Web"
+mock_collection_spoc_3["sponsored_by_override"] = "Brought to you by WallMarket"
 
 mock_collection = {
     "title": "Best of the Web",
     "sponsor": "WallMarket",
+    "sponsored_by_override": "Brought to you by WallMarket",
```

Comment:
I am not clear what these changes to fixtures would do, if anything. Do we need to add these for tests to pass? Because that's not what the code seems to imply.

Reply:
AdZerk doesn't have a concept of 'collections' that can have attributes, so the code is a bit of a hack. We set fields on individual spocs (lines 47, 50) and pull them up to the collection level (line 55). This comment tries to explain this:
```
def to_collection(spocs):
    """
    Transforms a list of spocs to a sponsored collection dictionary.
    AdZerk does not support fields for a collection. We set them on all creatives and get them from an arbitrary one.
```
Is there a comment I can place somewhere to clarify this better?